pkg/chunk/schema.go

```diff
@@ -529,6 +539,39 @@ func (entries v7Entries) GetWriteEntries(bucket Bucket, metricName model.LabelVa
 }
 
 func (v7Entries) GetReadQueries(bucket Bucket) ([]IndexQuery, error) {
+	// Replaced with v8Schema series index
+	return nil, ErrNoMetricNameNotSupported
+}
+
+// v8Entries is the same as v7Entries however with a series index instead of a metric name index
```

Comment:
`v8Entries is the same as v7Entries`
does not seem consistent with:
`type v8Entries struct { v6Entries }`
which one is correct? is just a typo in the comment?